DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -26,6 +24,9 @@ public CalculatorPage(IWebDriver driver)
         private IWebElement IncomeFld => driver.FindElement(By.XPath("//th[text()='Income: *']/..//input"));
         private IWebElement InterestEarnedFld => driver.FindElement(By.XPath("//th[text()='Intereset earned: *']/..//input"));
         private IWebElement EndDateFld => driver.FindElement(By.XPath("//th[text()='End date: *']/..//input"));
+        private IWebElement SettingsLnk => driver.FindElement(By.XPath("//div[@onclick='Settings()']"));
```

Comment:
I think it is better to use `text` in this case.
```suggestion
        private IWebElement SettingsLnk => driver.FindElement(By.XPath("//div[text() = 'Settings']"));
```